.github/workflows/python-package.yml

```diff
@@ -0,0 +1,37 @@
+name: Python package
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
would we want to run this on all branches?

Reply:
I think we probably want to encourage people to develop on forks (unlike what I'm doing right now), so most testing will be through pull requests on main, below.